.github/CODEOWNERS

```diff
@@ -816,7 +816,7 @@ sdk/dnsresolver/arm-dnsresolver @qiaozha @dw511214992 @MaryGao
 sdk/dnsresolver/ci.mgmt.yml @qiaozha @dw511214992 @MaryGao
 
 # PRLabel: %Monitor
-/sdk/monitor/ @hectorhdzg
+/sdk/monitor/ @hectorhdzg @JacksonWeber
```

Comment:
@jsquire @ronniegeraghty do you know how Jackson can request for write permissions?

Reply:
https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/49/Request-Access-to-Azure-SDK-repos